packages/editor/src/plugins/fill-in-the-blanks-exercise/editor.tsx

```diff
@@ -26,27 +37,66 @@ export function FillInTheBlanksExerciseEditor(
   })
```

Comment:
not your change, but I'd prefer a different name for `textPluginState` since it uses selectDocument. Maybe just `textDocument`